eng-paths/staff_engineer.md

```diff
@@ -88,14 +89,16 @@ Whole platform and organization
 <table  markdown="1">
 <tr><td>
 
-Sr Software Engineer 2
+Sr Software Engineer 3
 
 </td></tr>
 <tr><td  markdown="1">
 
-- Technical expert on 2 products / key components, solid on the rest of the platform.
+- Technical expert on multiple products / key components, knowing all the extent of their core team's domain.
```

Comment:
Same as above, staffs need to be solid on the entire platform, and don't have a core team anymore...

Reply:
ohh, that's not the staff one, that's the SE3 description appearing inside the staff file given the way we compile/show the data. Are we good here then? 🙂